examples/scala/src/main/scala/example/Streaming.scala

```diff
@@ -36,6 +36,7 @@ object Streaming {
 
     import spark.implicits._
 
+    println("=== Section 1: write and read delta table normally, and for Section 3")
```

Comment:
I'd replace "normally" with "using batch queries" and the last bit to "initialize table for later sections".

Reply:
Yes, I updated with your comment.